cmake/mayausdapi_version.info

```diff
@@ -1,3 +1,3 @@
-set(MAYAUSDAPI_MAJOR_VERSION 0)
+set(MAYAUSDAPI_MAJOR_VERSION 1)
```

Comment:
Now set to major version 0 - binary breaking changes are not allowed.

Reply:
This means that current binaries that depend on mayaUsdAPI v0 must be recompiled.

Reply:
Yes true. Is that going to be a problem? The only two that I know of that use the API are Hydra and LookdevX. I will update the MayaUsd cut used by LookdevX in my LDX PR (after this one is merged). Hydra will get the latest one the next time they update and we'll ship new 2025 binaries the next release.